capsules/src/usb/cdc.rs

```diff
@@ -416,29 +456,30 @@ impl<'a, U: hil::usb::UsbController<'a>, A: 'a + Alarm<'a>> hil::usb::Client<'a>
     /// Handle a Control Out transaction
     fn ctrl_out(&'a self, endpoint: usize, packet_bytes: u32) -> hil::usb::CtrlOutResult {
         // Check what state our Ctrl endpoint is in.
-        if self.ctrl_state.get() == CtrlState::SetLineCoding {
-            // We got a Ctrl SET_LINE_CODING setup, now we are getting the data.
-            // We can parse the data we got.
-            descriptors::CdcAcmSetLineCodingData::get(&self.client_ctrl.ctrl_buffer.buf).map(
-                |line_coding| {
-                    // Check if we should switch our main state machine to
-                    // connecting meaning that the host is connecting to the virtual
-                    // serial port. We decide this based on if the host is
-                    // configuring the baud rate to what we expect.
-                    if self.state.get() == State::Enumerated && line_coding.baud_rate == 115200 {
-                        self.state.set(State::Connecting);
-                    }
+        match self.ctrl_state.get() {
+            CtrlState::SetLineCoding => {
```

Comment:
Why change this from an `if`?

Reply:
I guess that is left over from some things I tried that didn't end up being necessary. But if someone does want to handle data from a different ctrl message they likely want to make this change.